packages/next/client/index.js

```diff
@@ -21,6 +21,9 @@ import { DataManager } from 'next-server/dist/lib/data-manager'
 if (!window.Promise) {
   window.Promise = Promise
```

Comment:
not related to this PR, but it caught my eye. Where is `Promise` defined?

Reply:
@Janpot I think it should be added by `babel`, @timneutkens correct me if I'm wrong 🙏

Reply:
currently its an inlined reference to `corejs/promise` as a result of babel